test_client.py

```diff
@@ -96,3 +116,26 @@ def calculate_reliability_score_for_client(self, msisdn: str) -> float:
         reliability = (successful_tests / total_tests) * 100
 
         return round(reliability, 2)
+
+    def timeout_tests(self):
+        """
+        Update the status of tests to 'timedout' if they are older than 10 minutes.
+
+        Returns:
+            int: The number of tests updated to 'timedout'.
+        """
+        try:
+            expiration_time = datetime.now() - timedelta(minutes=10)
+
+            timedout_tests_query = ReliabilityTests.update(status="timedout").where(
+                ReliabilityTests.start_time < expiration_time,
+                ReliabilityTests.status != "timedout",
```

Comment:
this should also check for `success` if it checks only for `timedout` it'll update `success` tests that have passed 10 mins